app/services/transaction_service/gateway/paypal_adapter.rb

```diff
@@ -19,8 +19,8 @@ def create_payment(tx:, gateway_fields:, prefer_async:)
         {
          transaction_id: tx[:id],
          item_name: tx[:listing_title],
-         item_quantity: 1,
-         item_price: item_total,
```

Comment:
Looks like this was the last reference to item_total. Remove variable + calculation.